viewing_party/party.py

```diff
@@ -1,23 +1,276 @@
 # ------------- WAVE 1 --------------------
+from collections import Counter
+from enum import unique
 
+# inputs a movie into the program
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+        return movie
+    else:
+        return None
+
+# takes a movie dict and adds it to user's watched
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data
+
+# takes a movie dict and adds it to user's watchlist
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data
+
+# finds the title of a movie in watchlist, then removes and transfers it to watched
+def watch_movie(user_data, title):
+    counter = 0
+    for movie in user_data["watchlist"]:
+        if title in movie["title"]:
+            user_data["watched"].append(movie)
+            del(user_data["watchlist"][counter])

```

Comment:
I don't believe `del` was covered in the material, so good job expanding your Python knowledge! I do think it's not widely used though it definitely works the way you're using it. `pop` and `remove` would be the most common, where the former is more useful for when you want to do something with the item it returns and latter is more useful when you know the item is in the list (which in this case we do) but don't need the reference to the item.